t/01.utils/go_figure_dbc.t

```diff
@@ -52,4 +53,8 @@ my $dba = Bio::EnsEMBL::Hive::DBSQL::DBAdaptor->new(-url => $pipeline_url,
 my $dba2dbc = go_figure_dbc($dba);
 isa_ok($dba2dbc, 'Bio::EnsEMBL::Hive::DBSQL::DBConnection');
 
+throws_ok {
+    Bio::EnsEMBL::Hive::DBSQL::DBAdaptor->new(-reg_conf => '/non_existent_file');
+} qr/Configuration file .* does not exist. Registry configuration not loaded/, 'Throws a relevant message if the path doesn\'t exist';
```

Comment:
Doesn't seem to match the actual error message - see failure on Travis for this test

Reply:
Yes, because it needs Ensembl/ensembl#408

Reply:
Actually, the real issue with the test is that we don't clone ensembl into Travis, therefore the `require Bio::EnsEMBL::Registry` fails.

Reply:
:man_facepalming:
I hope I can get away with just cloning ensembl, without having to install many other dependencies